src/include/sof/schedule/edf_schedule.h

```diff
@@ -20,6 +21,10 @@
 
 struct edf_task_pdata {
 	void *ctx;
+	uint32_t budget;	/**< cycles per systick */
```

Comment:
I am pretty sure in the past we defined variable ticks for EDF. Would this scale here if the tick duration is changed dynamically. And I read in a comment below that we use '120k cycles in systick instead of 400k', so shouldn't there be multiple budget definitions, one per systick value?